supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -189,20 +223,36 @@ Always use "Administration Portal", including the capital P. When other departme
 [discrete]
 [[AMD64]]
 ==== image:images/yes.png[yes] AMD64 (noun)
-*Description*: _AMD64_ is the AMD implementation of a 64-bit version of the x86 architecture.
+*Description*: AMD's 64-bit version of the x86 architecture. Use this format for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, CLI objects. Use this format in general sentences when referring to OCP features. 
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_This Operator is supported on AMD64 and ARM64 platforms._ 
 
 *Use it*: yes
 
-*Incorrect forms*: Hammer, x86_64, x86-64, x64, 64-bit x86
+*Incorrect forms*: _AMD64_ 
 
 *See also*:
 
-[NOTE]
-====
-The AMD64 logo is trademarked; the term _AMD64_ is not trademarked. For more information about AMD trademarks, see the http://www.amd.com/us/aboutamd/Pages/trademarks.aspx[AMD Trademark Information] page.
+[discrete]
+[[amd64]]
+==== image:images/yes.png[yes] amd64 (noun)
+*Description*: AMD's 64-bit version of the x86 architecture. Use this term for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, CLI objects. Use lowercase (and backticks) when referring to objects or parameters.
```

Comment:
```suggestion
*Description*: AMD's 64-bit version of the x86 architecture. Use this term for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, and CLI objects. Use lowercase (and backticks) when referring to objects or parameters.
```

Reply:
Should be _Operators_ capitalized?

And I would remove the brackets around _and backticks_.

Reply:
Yes we capitalize Operators in OCP docs